CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxForwardSpeed: 10.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxReverseSpeed: 10.0
CarFreedom_Config_CarFreedom_MaxSteerAngleLeft: -20.1607
CarFreedom_Config_CarFreedom_MaxSteerAngleRight: +20.1246
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxSteerAngleAtMaxSpeed: 15.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxFowardAcceleration: 4.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxReverseAcceleration: 4.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxSteerAngleRightAcceleration: 4.0
CarFreedom_Config_VehicleDescription_Control_VelocityBicycle_MaxSteerAngleLeftAcceleration: 4.0
Carfreedom_controle_watchdog_duration: 2.0
